numerics/fast_fourier_transform.hpp

```diff
@@ -21,46 +21,59 @@ using base::FloorLog2;
 using geometry::Complexification;
 using geometry::Hilbert;
 using geometry::Interval;
-using quantities::AngularFrequency;
-using quantities::Time;
-
-// This class computes Fourier[{...}, FourierParameters -> {1, -1}] in
-// Mathematica notation (the "signal processing" Fourier transform).
-template<typename Value, std::size_t size_>
+using quantities::Angle;
+using quantities::Derivative;
+using quantities::Difference;
+
+// Given (u₀, ..., uₙ), this class computes the discrete Fourier transform
+//   Uₛ = ∑ᵣ uᵣ exp(-2πirs/n),
+// corresponding to
+//   Fourier[{...}, FourierParameters -> {1, -1}]
+// in Mathematica notation (the "signal processing" Fourier transform).
+template<typename Value, typename Argument, std::size_t size_>
 class FastFourierTransform {
  public:
+  using AngularFrequency = Derivative<Angle, Argument>;
```

Comment:
Document that this is only a *bona fide* angular frequency if `Argument` is time-like.